locales/fr_FR/pcsx2_Main.po

```diff
@@ -2221,7 +2215,7 @@ msgstr "Taille de la fenêtre :"
 
 #: pcsx2/gui/Panels/GSWindowPanel.cpp:109
 msgid "Zoom:"
-msgstr ""
+msgstr "Zoom :"
```

Comment:
Remove space before :

Reply:
French language has spaces before and after theses characters:  ? ! : ; % " + - / * =
See: https://www.blogdumoderateur.com/regles-ponctuation-typographie/